hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/web/JsonUtil.java

```diff
@@ -725,4 +726,21 @@ public static Map<String, Object> toJsonMap(BlockLocation[] locations)
     m.put(BlockLocation.class.getSimpleName(), blockLocations);
     return m;
   }
+
+  public static String toJsonString(FsStatus status)
+      throws IOException {
+    return toJsonString(FsStatus.class, toJsonMap(status));
+  }
+
+  public static Map<String, Object> toJsonMap(FsStatus status)
+      throws IOException {
```

Comment:
this doesn't throw IOE, can be removed, once you remove it from here, I think the throws IOE can be removed from the above method as well